### ISSUE TITLE
E-mail initiated chat

### ISSUE THREAD

**Issue by MrSaints** [papercups-io/papercups#131]
Awesome work on the launch! 

I was wondering if y'all have any thoughts on or plans on supporting chats initiated by e-mail?

There already seems to be plans for continuing a chat through e-mail (https://github.com/papercups-io/papercups/wiki/Roadmap#after-hours), so this seems like a nice enhancement to it.

**Comment by reichert621** [comment]
hey @MrSaints! can you elaborate a bit more on how you would like this to work?

would your customer send an email to a special address you specify, and then have this show up in the papercups dashboard so you can reply there?

**Comment by MrSaints** [comment]
> would your customer send an email to a special address you specify, and then have this show up in the papercups dashboard so you can reply there?

Hey @reichert621, yes, that's exactly what I had in mind :)

**Comment by reichert621** [comment]
makes sense! we definitely want to prioritize our email integrations since we've been getting a lot of feedback about how important that is to people :)
